package.nls.de.json

```diff
@@ -1,4 +1,5 @@
 {
+  "command.activateIcons.title": "",
```

Comment:
`Dateisymboldesign ` is really shitty. We say `icons` really natively... I don't know why the translators at transifix like to use this "word"... 🤢